src/gui/qgsmaptip.cpp

```diff
@@ -146,7 +146,7 @@ void QgsMapTip::showMapTip( QgsMapLayer *pLayer,
 
   QgsDebugMsg( tipHtml );
 
-  mWidget->move( pixelPosition.x(),
+  mWidget->move( pixelPosition.x() + 20,
```

Comment:
I don't think this will work on hidpi -- you'll probably need to do something like the cursor scaling itself:

    float scale = Qgis::UI_SCALE_FACTOR * fontMetrics().height() / 32.0;

Reply:
Indeed, I was expecting that. So, I was hoping for some insight into how that could be done.